mlflow/metrics/genai/metric_definitions.py

```diff
@@ -0,0 +1,57 @@
+from typing import List, Optional
+
+from mlflow.exceptions import MlflowException
+from mlflow.metrics.base import EvaluationExample
+from mlflow.metrics.genai.make_genai_metric import make_genai_metric
+from mlflow.models import EvaluationMetric
+from mlflow.protos.databricks_pb2 import INTERNAL_ERROR, INVALID_PARAMETER_VALUE
+from mlflow.utils.class_utils import _get_class_from_string
+
+DEFAULT_MODEL = "openai:/gpt4"
+LATEST_VERSION = "v1"
```

Comment:
can we create a helper in the utils folder to get the latest version. Right now the helper returns back a hardcoded string: v1, but in future we can update the helper to be start enough to return the latest version. 

And we can use this help in make_genai_metric file as well so its shared